src/components/FButton.vue

```diff
@@ -554,7 +553,9 @@ const style = computed(
   })
 );
 
-function handleClick(e: MouseEvent | KeyboardEvent, blur: boolean): void {
+function handleClick(e: PointerEvent, blur: boolean): void {
+  if (!e.pointerType) return;
```

Comment:
1) add a comment (because if we are not familiar with pointer events it cannot be understood easily)
2) I don't understand, if you return, you never emit the click event to the parent...